gulpfile.js

```diff
@@ -144,6 +144,18 @@ gulp.task(
   )
 )
 
+gulp.task(
+  'copy:govuk_frontend:stylesheets',
+  function () {
+    return gulp
+      .src(path.join(govukFrontendRoot, '**', '*.scss'), { base: govukFrontendRoot })
+      .pipe(gulp.dest(path.join('app', 'assets', 'scss')))
+      .on('end', function () {
+        console.log('📂  Copied stylesheets from GOV.UK Frontend')
+      })
+  }
+)
+
```

Comment:
Instead of doing this we could add `node_modules/govuk-frontend` to the [include paths in the gulpfile]( https://github.com/alphagov/digitalmarketplace-user-frontend/blob/master/gulpfile.js#L34).

Which do you think is better?

Reply:
include paths!